CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ ELSE()
     IF(${PACKAGE_NAME}_TRILINOS_TPL)
         LIST(APPEND CMAKE_PREFIX_PATH ${TPL_Trilinos_LIBRARY_DIRS}/.. /usr/local/cuda/lib64)
 
-        find_package(Kokkos REQUIRED QUIET)
+        find_package(Kokkos REQUIRED QUIET HINTS ${Trilinos_DIR}/../Kokkos)
```

Comment:
This is not needed for the Spack build of DTK with older versions of Trilinos, is it?  Spack should be setting the Trilinos install dir in `CMAKE_PREFIX_PATH` where KokkosConfig.cmake will be found automatically.

Reply:
Our CI which only sets `TPL_Trilinos_DIR` doesn't find `Kokkos`, though.